applications_tests/dem_3d/rotating_in_ball.prm

```diff
@@ -91,11 +91,11 @@ subsection mesh
 end
 
 #---------------------------------------------------
-# Boundary Motion
+#  DEM Boundary Conditions
 #---------------------------------------------------
-subsection boundary motion
-  set number of boundary motion         		= 1
-    subsection moving boundary 0
+subsection DEM boundary conditions
+  set number of boundary conditions         		= 1
```

Comment:
Why is there only one boundary condition set? does the default behavior is to assume all boundaries without anything assigned to them are walls?

Reply:
That's exactly that. And if the user wanna still put a boundary condition as wall, the option exists as well. (as default)

Reply:
Yes, as @EduardsBlandin mentioned, the default condition for the boundaries is fixed_wall.